Lecture 3/Task1/AircraftCompare.java

```diff
@@ -0,0 +1,11 @@
+
+import java.util.Comparator;
+
+public class AircraftCompare implements Comparator<Aircraft> {
+
+    @Override
+    public int compare(Aircraft a1, Aircraft a2) {
+        return new Integer(a1.getFlightDistance()).compareTo(a2.getFlightDistance());

```

Comment:
what if flight distance is the same? is it is the same aircraft? extend this method to really compare two aircrafts by all the params available.